xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererMediaCodec.cpp

```diff
@@ -116,8 +116,8 @@ CRenderInfo CRendererMediaCodec::GetRenderInfo()
 {
   CRenderInfo info;
   info.formats = m_formats;
-  info.max_buffer_size = NUM_BUFFERS;
-  info.optimal_buffer_size = 2;
+  info.max_buffer_size = 4;
```

Comment:
I'd add that max_buffer_size has no purpose, thus, IMO, unless you accept that optimal might be > max?